horreum-backend/src/main/java/io/hyperfoil/tools/horreum/server/EncryptionManager.java

```diff
@@ -1,66 +0,0 @@
-package io.hyperfoil.tools.horreum.server;
-
-import java.nio.charset.StandardCharsets;
-import java.security.GeneralSecurityException;
-import java.security.NoSuchAlgorithmException;
-import java.security.SecureRandom;
-import java.security.spec.InvalidKeySpecException;
-import java.security.spec.KeySpec;
-import java.util.Base64;
-
-import javax.crypto.Cipher;
-import javax.crypto.SecretKey;
-import javax.crypto.SecretKeyFactory;
-import javax.crypto.spec.GCMParameterSpec;
-import javax.crypto.spec.PBEKeySpec;
-import javax.crypto.spec.SecretKeySpec;
-
-import jakarta.annotation.PostConstruct;
-import jakarta.enterprise.context.ApplicationScoped;
-
-import org.eclipse.microprofile.config.inject.ConfigProperty;
-
-@ApplicationScoped
-public class EncryptionManager {
-    @ConfigProperty(name = "horreum.db.secret")
```

Comment:
Can we finally remove `horreum.db.secret` everywhere? iirc, tokens were the last part of the system still using this property. 

Although it looks like the DB table has been dropped for a while, we can remove horreum.db.secret everywhere is it mentioned in docs, and scripts

Reply:
+1 I think that could have been removed already but I agree to remove all references in this PR directly